script.js

```diff
@@ -170,3 +170,31 @@ form.addEventListener('submit', (e) => {
   validateInputs();
 });
 // #endregion
+// Get All input fields and text area in the doc
+const inputFields = document.querySelectorAll('input, textarea');
+// Loop through each input field and add an "input" event listener
+inputFields.forEach((input) => {
+  input.addEventListener('input', (event) => {
+    // Get the updated value of the input field
+    const updatedValue = event.target.value;
+    // Get the key name for the local storage item (use the input field's name attribute)
+    const keyName = event.target.name;
+    // Save the updated value to local storage
+    localStorage.setItem(keyName, updatedValue);
+  });
+});
+
+// Add an event listener for the "DOMContentLoaded" event
+document.addEventListener('DOMContentLoaded', () => {
+  // Loop through each input and textarea field
+  inputFields.forEach((input) => {
+    // Get the key name for the local storage item (use the input field's name attribute)
+    const keyName = input.name;
+    // Get the saved value from local storage (if it exists)
+    const savedValue = localStorage.getItem(keyName);
+    // If a saved value exists, set the value of the input or textarea field to the saved value
+    if (savedValue) {
+      input.value = savedValue;
+    }
+  });
+});
```

Comment:
- [ ] When the user loads the page, the code checks if there's any saved data in the local storage and if so the input fields are pre-filled with the data Just like the below sample. 
```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};

```

Reply:
@Abenezer-Tilahun thank you for the thoughtful review, I have implemented the changes but I can't submit it again on my dashboard: 
<img width="1007" alt="image" src="https://user-images.githubusercontent.com/115372699/224437459-419f90fc-7e91-4024-8296-8e5fa8fa0590.png">
it shows as "review pending", do you need to do something on your end to mark this review as done so I can send it to be reviewed again?